content/docs/user-guide/pipelines/index.md

```diff
@@ -1,19 +0,0 @@
-# Pipelines
-
-If you find yourself repeating sequence of actions to get or update the results
-of your project, then you may already have a pipeline. For example, a data
-science workflow could involve:
-
-1. Gathering data for training and validation
-2. Extracting useful features from the training dataset
-3. (Re)training an ML model
-4. Evaluating the results against the validation set
```

Comment:
Up to you but [this](https://dvc.org/doc/user-guide/pipelines) seemed like a nice intro. Keep in mind we plan to add more pages to this section (see https://github.com/iterative/dvc.org/issues/2883).

Reply:
Sorry, this was a mistake from when I initially got rid of the pipelines guide. Adding back ⏳